source/opt/basic_block.cpp

```diff
@@ -23,9 +23,8 @@ BasicBlock::BasicBlock(const BasicBlock& bb)
     : function_(nullptr),
       label_(MakeUnique<Instruction>(bb.GetLabelInst())),
       insts_() {
-  insts_.reserve(bb.insts_.size());
   for (auto& inst : bb.insts_)
-    AddInstruction(MakeUnique<Instruction>(*inst.get()));
+    AddInstruction(std::unique_ptr<Instruction>(inst.Clone()));
```

Comment:
I made the change to avoid using the copy constructor that MakeUnique uses.  The new code works with all of the compiler that are use in our continuous builds.